master/reference/repo-structure.md

```diff
@@ -1,69 +1,63 @@
-  Docker image which is used to build the `calicoctl` binary.
-- UTs testing calicoctl, STs testing single host and multihost systems
-  using calicoctl and calico/node to create Calico networked containers.  These
-  tests run within the `calico/test` Docker image.
-  for the UT and ST codebase.  Also see the Makefile for
-  details on how the STs (`make st`) and UTs (`make ut`) are run using the
-  `calico/test` image.
-- Release scripts used to validate our documentation and to cut a new branch
-  for a specific release.
-
-It pulls together a number of related repositories that provide subsets of
-function for running Calico in a containerized environment.  These related
-respositories are listed below.
-
-## Calico Libraries
-
- - [calico](https://github.com/projectcalico/calico): This repo contains the
-   implementation of Felix which is the heart of Calico networking.  Felix
-   interfaces with the Linux kernel to configure routes and ACLs
-   that control network policy connectivity. In `calico/node`, Felix runs as a
-   separate process.  It is installed directly into the `calico/node` image.
-
- - [libcalico](https://github.com/projectcalico/libcalico): Contains a variety
-   of helper methods and classes for manipulating the Calico data in an etcd
-   datastore, IP address management and useful namespace utilities to
-   manipulating container interfaces.  The libcalico library is used by
-   `calicoctl`.  It is imported as a standard Python library into the
-   `calicoctl` Python codebase.  libcalico also contains the `calico/test` Docker
-   image used for running both UTs and STs testing Calico with Docker.
-
-## Calico Orchestrator Integration Plugins
+## Core Components
+
+### [Felix](https://github.com/projectcalico/felix)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `felix` | Binary |The heart of Calico networking, Felix interfaces with the Linux kernel to configure routes and ACLs that control network policy connectivity. Felix is a common component run in every orchestration system.|
+
+
+### [Calico-Containers](https://github.com/projectcalico/calico-containers)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `calico/node`| Docker Image |  The Docker image used to run the Calico Felix agent (responsible for IP routing and ACL programming) and a BIRD BGP agent (for distributing routes between hosts) for Containerized workloads (i.e. not VMs with Openstack).  See [`calico_node` directory]({{site.baseurl}}/{{page.version}}/reference/architecture) for details. |
+| `calicoctl`| Binary | The command line tool used for starting a `calico/node`  container, configuring Calico policy, adding and removing containers in a  Calico network via orchestration tools, and managing certain network and  diagnostic administration.  See [`calicoctl` directory]({{site.baseurl}}/{{page.version}}/reference/calicoctl) for implementation of calicoctl. |
```

Comment:
> The command line tool used for starting a `calico/node`  container, configuring Calico policy, adding and removing containers in a  Calico network via orchestration tools

Don't need to go into all of this detail. e.g "The command line tool for interacting with the Calico API".

> The command line tool used for starting a `calico/node`

I don't think we  want to make this the front-and-center feature of calicoctl.  calicoctl should be positioned mostly as interacting with the Calico API, and oh yeah it also provides some helpful tools like diags, etc"